src/yamlfix/adapters.py

```diff
@@ -606,6 +626,47 @@ def _fix_comments(self, source_code: str) -> str:
 
         return "\n".join(fixed_source_lines)
 
+    def _fix_whitelines(self, source_code: str) -> str:
+        r"""Fixes number of consecutive whitelines.
```

Comment:
Possible typo `r"""` -> `"""`

Reply:
When replacing `r"""` by `"""`, pylint raises this error:
```
D301 Use r""" if any backslashes in a docstring [flake8-docstrings]
```